lib/mayaUsd/fileio/translators/translatorGprim.cpp

```diff
@@ -37,12 +37,17 @@ void UsdMayaTranslatorGprim::Read(
 void UsdMayaTranslatorGprim::Write(
     const MObject&      mayaNode,
     const UsdGeomGprim& gprim,
-    UsdMayaPrimWriterContext*)
+    UsdMayaPrimWriterContext*,
+    GeomSidedness sidedness)
 {
     MFnDependencyNode depFn(mayaNode);
 
-    bool doubleSided = false;
-    if (UsdMayaUtil::getPlugValue(depFn, "doubleSided", &doubleSided)) {
+    bool doubleSided = sidedness == GeomSidedness::Double;
+    if (sidedness == GeomSidedness::Derived) {
+        if (UsdMayaUtil::getPlugValue(depFn, "doubleSided", &doubleSided)) {
```

Comment:
Am I missing something here?  UsdMayaUtil::getPlugValue() returns true for a successful operation.  The value of the plug is in doubleSided.  Therefore, shouldn't this read
if (UsdMayaUtil::getPlugValue(depFn, "doubleSided", &doubleSided) && doubleSided) 
?

Reply:
I'm trying to preserve the existing behavior where it always writes the attribute, regardless of if it's true or false.
I could change it to only write it if it's true, but there shouldn't be a downside to it one way or another.

Reply:
Thanks for the explanation; it would make a great code comment.

Reply:
Added a comment to that block